plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/CabalFields.hs

```diff
@@ -1,12 +1,16 @@
-module Ide.Plugin.Cabal.Completion.CabalFields (findStanzaForColumn, findFieldSection, getOptionalSectionName, getAnnotation, getFieldName, onelineSectionArgs) where
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+module Ide.Plugin.Cabal.Completion.CabalFields (findStanzaForColumn, findFieldSection, findTextWord, findFieldLine, getOptionalSectionName, getAnnotation, getFieldName, onelineSectionArgs, getFieldEndPosition, getSectionArgEndPosition, getNameEndPosition, getFieldLineEndPosition, getFieldLSPRange) where
 
+import qualified Data.ByteString                   as BS
+import           Data.List                         (find)
 import           Data.List.NonEmpty                (NonEmpty)
 import qualified Data.List.NonEmpty                as NE
 import qualified Data.Text                         as T
 import qualified Data.Text.Encoding                as T
 import qualified Distribution.Fields               as Syntax
 import qualified Distribution.Parsec.Position      as Syntax
 import           Ide.Plugin.Cabal.Completion.Types
+import qualified Language.LSP.Protocol.Types       as LSPTypes
```

Comment:
Nitpick:
```suggestion
import qualified Language.LSP.Protocol.Types       as LSP
```

Other parts of the codebase use this qualified import. Perhaps we should have some kind of style guide? Hard to tell.